solutions/src/main/java/ru/sberbank/school/task06/CountMapImpl.java

```diff
@@ -0,0 +1,66 @@
+package ru.sberbank.school.task06;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CountMapImpl<T> implements CountMap<T> {
+
+    private final Map<T, Integer> map;
+
+    public CountMapImpl() {
+        this.map = new HashMap<>();
+    }
+
+    @Override
+    public void add(T o) {
+        Integer count = map.get(o);
+        map.put(o, count == null ? 1 : count + 1);
+    }
+
+    public int getCount(T o) {
+        try {
+            return map.get(o);
+        } catch (NullPointerException ex){
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a212674429>*